### ISSUE TITLE
Souce Error with every video on every instance. Video and Audio does not play.

### ISSUE THREAD

**Issue by goose-in-ranch** [libre-tube/LibreTube#5875]
### Steps to reproduce

I open a video, and the video does not load or play. (Title, viewcount, upload date, recommendations with thumbnails load fine). I switch to audio and am met with "Source Error" toaster message. Switching instances results in same behavior.

https://github.com/libre-tube/LibreTube/assets/80707474/297905cc-b002-441a-a994-2f9c332e5eb5

### Expected behavior

Video should play when clicked on. The video's audio should play when I click on audio.

### Actual behavior

Video does not load, and does not play. Audio does not load, and does not play.

### LibreTube version

v0.22.2 & v0.22.1

### Android version

Android 14

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Ehab-d** [comment]
This is not an instance problem. Switch from DASH to HLS as There are problems with DASH and Piped currently. 
`Settings -> Audio and video -> Quality -> Use HLS`
The issue is tracked in #5855.

**Comment by Ehab-d** [comment]
The issue is tracked in #5855, so this is a duplicate.

**Comment by goose-in-ranch** [comment]
I just switched "Use HLS" on and I am getting the same behavior.

**Comment by Bnyro** [comment]
https://github.com/libre-tube/LibreTube/issues/5855

**Comment by Ehab-d** [comment]
> Enabling this, then switching instances solved video! Switching to audio still leads to "source error" toaster message however. Do not know if this is related to HLS.

Yes. The audio player did not respect the `Use HLS` setting and always used DASH. This issue was resolved in #5862. You can expect it to be resolved in the next release or try the [nightly builds](https://github.com/libre-tube/NightlyBuilds).
